users/src/routes/team.routes.ts

```diff
@@ -18,19 +17,11 @@ router.get('/members/:name', requireUser, teamController.getTeamMember);
 router.post('/', requireUser, createTeamValidator, validateRequest, teamController.createTeam);
 
 router.post(
-  '/members',
+  '/invite-member',
```

Comment:
we need to discuss naming route, I think team and connection need to be consistent